lib/ConfigManager/ConfigManager.js

```diff
@@ -13,7 +13,7 @@ class ConfigManager extends React.Component {
       type: 'okapi',
       path: (_q, _p, _r, _l, props) => (props.moduleName ?
         `configurations/entries?query=(module==${props.moduleName} and configName==${props.configName})` :
-        `settings/entries?query=(scope==${props.scope} and key==${props.configName})`
+        `settings/entries?query=(scope==${props.scope} and key==${props.configName}${props.userId ? ` and userId==${props.userId}` : ''})`
```

Comment:
Minor but this template is reaching a point of complexity where I would break them out into separate functions/variables with differentiated naming - `getConfigurationsPath()`... `getSettingsPath()` etc...